components/auth/Registration/RegistrationForm.module.scss

```diff
@@ -27,10 +27,8 @@
       }
     }
     h1 {
-      color: #fff;
+      @include s.h1;
```

Comment:
your "s" don`t describe itself, use imagination to name it. We are working in team , all of us need to understand what you are including here,  without diving into file